### PR TITLE
Migrate to new Runtime APIs

### PR DESCRIPTION
**Pull request by ravinperera00**
## Checklist
- [ ] Linked to an issue
- [ ] Updated the specification
- [ ] Updated the changelog
- [ ] Added tests
- [ ] Checked native-image compatibility

### CODE REVIEW

File: native/src/main/java/io/ballerina/stdlib/websocket/ReturnStreamUnitCallBack.java [hunk 1]
```diff
@@ -72,8 +72,7 @@ public void notifySuccess(Object response) {
                 sendTextMessageStream(StringUtils.fromString(content), promiseCombiner);
                 Thread.startVirtualThread(() -> {
                     try {
-                        Object result = runtime.startIsolatedWorker(bObject, STREAMING_NEXT_FUNCTION, null,
-                                null, null).get();
+                        Object result = runtime.callMethod(bObject, STREAMING_NEXT_FUNCTION, null);
```
Comment (warunalakshitha) [inline native/src/main/java/io/ballerina/stdlib/websocket/ReturnStreamUnitCallBack.java:75]: We should pass strandmetadata with concurrent safe true

Comment (warunalakshitha) [reply native/src/main/java/io/ballerina/stdlib/websocket/ReturnStreamUnitCallBack.java:75]: Check in other places too

Comment (ravinperera00) [reply native/src/main/java/io/ballerina/stdlib/websocket/ReturnStreamUnitCallBack.java:75]: Will fix them

Comment (ravinperera00) [reply native/src/main/java/io/ballerina/stdlib/websocket/ReturnStreamUnitCallBack.java]: Fixed in ad61731008116e64e97785cca3b3c54cbd6b21b0

**Review by warunalakshitha — APPROVED** [review]

**Pull request merged**